### ISSUE TITLE
ZefyrField : hintText is coming at the bottom of the field.

### ISSUE THREAD

**Issue by ropali** [memspace/zefyr#163]
I have used this piece of code which is most of the parts are taken from one of your medium blog posts.
```
@override
  Widget build(BuildContext context) {

    double screenHeight = MediaQuery.of(context).size.height;

    double editorHeight = screenHeight * 0.65;

    final editor = ZefyrField(
      height: editorHeight, // set the editor's height
      controller: _controller,
      focusNode: _focusNode,
      autofocus: false,
      decoration: InputDecoration(
        hintText: 'Enter Title Here...',
        border: OutlineInputBorder(
          borderRadius: BorderRadius.circular(5.0),
        ),
      ),
      physics: ClampingScrollPhysics(),
    );
    final form = Padding(
        padding: EdgeInsets.all(10),
        child: SingleChildScrollView(
          child: Column(
            children: <Widget>[
              TextField(
                decoration: InputDecoration(
                  hintText: 'Enter Title Here...',
                  border: OutlineInputBorder(
                    borderRadius: BorderRadius.circular(5.0),
                  ),
                ),
              ),
              SizedBox(height: 10,),
              editor
            ],
          ),
        ));

    return Scaffold(
        appBar: AppBar(
          title: Text(
            "Create New Notes",
            style: TextStyle(color: Colors.white),
          ),
          centerTitle: true,
          leading: IconButton(
            icon: Icon(Icons.chevron_left),
            color: Colors.white,
            onPressed: () => Navigator.pop(context),
          ),
          actions: <Widget>[
            Builder(
              builder: (context) => IconButton(
                  icon: Icon(Icons.save),
                  color: Colors.white,
                  onPressed: () => _saveDocument(context)),
            ),
          ],
        ),
        body: ZefyrScaffold(child: form));
  }
```
Now the hintText is coming at the bottom of the field which is guess not supposed be there. Look at the image below,

![image](https://user-images.githubusercontent.com/31515245/64795574-f0510700-d59b-11e9-9990-4f2f63024d82.png)
Is there any way to resolve this?

**Comment by pulyaevskiy** [comment]
Thanks for reporting! I spent some time trying to figure out what’s causing this, but don’t have a solution yet.

It does work in regular TextField with maxLines > 1, but I couldn’t locate any difference that would remotely cause this. Will try to do another round later.

**Comment by wangbo4020** [comment]
I have the same problem.

**Comment by TutOsirisOm** [comment]
Same issue here

**Comment by pulyaevskiy** [comment]
This is fixed in the 1.0-dev branch and will be included in the first dev release of Zefyr 1.0. Closing for now. Will update when Zefyr 1.0-dev.1 is out.
